### DISCUSSION TITLE
Using frameworks

### DISCUSSION THREAD

**Discussion by JunaidMancheri** [monkeytypegame/monkeytype#4073] [Ideas] [answered]
Upon inspecting the codebase, I noticed that no popular front-end framework was used. I am curious to know the reason behind this choice, and whether there are any plans to incorporate frameworks like React or Angular in the future.

**Comment by Miodec** [comment, marked as answer]
The reason is that's just how I got used to doing things. Keep in mind that I did not plan for the website to get so popular and that I have not worked professionally in web development before this. No plans to use any frameworks for now.

**Reply by JunaidMancheri** [reply]
Do Inform us if you plan somewhere in the future, I would like to collab.

**Comment by unrealapex** [comment]
Monkeytype does use jQuery.

**Reply by Miodec** [reply]
I think i would call it more of a library than a framework
